Cargo.toml

```diff
@@ -58,6 +58,7 @@ itertools = "0.10"
 pin-project = "1.0"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
+serde_str = "0.1.0"
```

Comment:
Once you enabled optional feature `serde-str` in `rust_decimal`.
We don't need to use `serde_str` here.